test/sanity/code-smell/boilerplate.json

```diff
@@ -0,0 +1,3 @@
+{
+    "disabled": true
+}
```

Comment:
Deleted tests should only have documentation left behind. No stub test or config file is required.

The docs will need to be marked as an orphan though:

```
:orphan:
```

Reply:
Done.